### PR TITLE
Feat #77: Added FAQ area

### PR DESCRIPTION
**Pull request by MigDinny**
(currently with dummy data)

Feat #77

Please Add Screenshots If any UI changes.

image

image

Replaced the menu in profile with a 3-dot button displaying a drop-down with 3 options. FAQ, Edit Profile and Settings.

Created FAQ Activity with several questions/answers displayed in dropdowns as well (ExpandableListViews).

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [ ] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [ ] If you have multiple commits please combine them into one commit by squashing them.

Note: gradle build failed but its errors have nothing to do with this PR - it must be handled in another PR.

### CODE REVIEW

**Comment by MigDinny** [comment]
@ankurs287 

Thanks! I've wrote a note in the original post, I don't know if you have read it.

I've run a gradlew check in the master branch and the build fails. I think those errors must be handled in a separate PR. I can try to do it myself. What do you think?

**Comment by ankurs287** [comment]
@MigDinny You need to update your local master branch as the gradle error has been fixed and working fine now.

**Comment by MigDinny** [comment]
image

image

image

master branch is up to date with upstream's. gradlew check on master reports those errors.

I actually did **update** my master branch before posting this PR... so, the problem is with master branch. try to build in your computer, it should produce the same errors i think.

@ankurs287

**Comment by letelete** [comment]
Also, can you upload your finbugs.html file? (It can be found at /Mifos/mobile-wallet/core/builds/reports/findbugs/findbugs.html) Just host it here: https://nofile.io

**Comment by MigDinny** [comment]
@letelete 
https://nofile.io/f/fcbb8WjQ2dv/FindBugs+Report.html

here it is

**Comment by letelete** [comment]
Your link gives me 404 error 

image

**Comment by MigDinny** [comment]
@letelete 
https://files.fm/u/xcq94nkm

had to use another service

that one is giving me that error everytime I upload a file

computers do not like me apparently

**Comment by MigDinny** [comment]
image

_one of the errors findbug.html is giving me_


image

_my code in my updated master's branch_

image

_upstream's code_


As you can see, they are the same.

**Comment by MigDinny** [comment]
@ankurs287 check this out! :)

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+            @Override
+            public void onGroupExpand(int groupPosition) {
+                Toast.makeText(getApplicationContext(),
+                        listDataGroup.get(groupPosition) + "Collapsed",
+                        Toast.LENGTH_SHORT).show();
+            }
+        });
+
+        // ExpandableListView Group collapsed listener
+        expandableListView.setOnGroupCollapseListener(new ExpandableListView.OnGroupCollapseListener() {
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java]: This line is has more than 100 characters, just fix this and it should work.

Comment (MigDinny) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java]: Will do it in a few minutes.

But the gradlew check i've performed in my computer reported errors in the master branch, how will this fix that?

Comment (luckyman20) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java]: Ignore that as of now, focus on this PR. After you get your PR merged, delete your fork and fork again and it should work.

**Comment by MigDinny** [comment]
@luckyman20 done

**Comment by ankurs287** [comment]
@MigDinny I re-ran the check. It is still getting failed.

**Comment by luckyman20** [comment]
@MigDinny See this: https://travis-ci.com/openMF/mobile-wallet/builds/91474835

It will tell you why your PR build is failing.

**Comment by MigDinny** [comment]
image

the errors are in files i never touched on. my branch is completely even with upstream's master. I cannot do nothing.

**Comment by MigDinny** [comment]
Check my screenshots. Those are the errors I’m getting in my computer. They are not the same Travis is reporting

**Comment by ankurs287** [comment]
@letelete Can you please help @MigDinny with the gradle errors as you fixed the issue before?

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java [hunk 2]
```diff
@@ -0,0 +1,164 @@
+package org.mifos.mobilewallet.mifospay.faq.ui;
+
+import android.os.Bundle;
+import android.view.View;
```
Comment (letelete) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java]: Remove unused import

**Comment by MigDinny** [comment]
@letelete 

Done.

image

I'm still getting failures in building the project. Can you pull my branch and check it in your computer? This is probably a local problem. I'll probably clone again after this PR is merged

**Comment by MigDinny** [comment]
@letelete oh it worked! :D thanks mate

**Pull request merged**